src/cogs/utils.py

```diff
@@ -17,6 +17,13 @@ class Utils(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+    @commands.command(name="ping")
```

Comment:
```suggestion
    @commands.command()
```
Pretty sure that's not needed.